android/titanium/src/java/org/appcelerator/titanium/view/TiUIView.java

```diff
@@ -1141,6 +1141,16 @@ public void processProperties(KrollDict d)
 			}
 		}
 
+		if (d.containsKey(TiC.PROPERTY_CLIP_MODE) && !nativeViewNull) {
+			if (nativeView != null
+				&& TiConvert.toInt(d, TiC.PROPERTY_CLIP_MODE) == -1
+				&& nativeView instanceof ViewGroup viewGroup
+				&& viewGroup.getParent() == null) {
+				viewGroup.setClipChildren(false);
+				viewGroup.setClipToPadding(false);
```

Comment:
- `setClipToPadding` only works if the view has a padding set, which we don't support at this moment.
- This may even work unexpected inside ListView. `setClipToChildren` and `setClipToPadding` serves entirely different purposes.
- In iOS, `clipsToBounds` is equivalent to `setClipToChildren`, but there's no direct equivalent to Android's `setClipToPadding`.

Reply:
Ah, good to know. I only used both because of the example module. I can remove the padding one